src/radical/pilot/agent/agent_0.py

```diff
@@ -41,22 +43,22 @@ class Agent_0(rpu.Worker):
     def __init__(self, cfg, session):
 
         self._uid     = 'agent.0'
-        self._cfg     = cfg
+        self._cfg: ru.Config = cfg
         self._pid     = cfg.pid
         self._pmgr    = cfg.pmgr
         self._pwd     = cfg.pilot_sandbox
+
         self._session = session
         self._log     = ru.Logger(self._uid, ns='radical.pilot')
-
+        self._log.info("LoggingConfig %s",self._cfg)
```

Comment:
The cfg is too large to log every time.

Reply:
made to debug. lmk if removing is required

Reply:
The log is stored in the filesystem anyway (`agent.0.cfg`), so logging it here is not needed.